widget/playground/src/containers/StyleLayout/StyleLayout.CustomColors.tsx

```diff
@@ -44,18 +47,33 @@ const Colors = (props: { mainColor?: string; secondColor?: string }) => {
 };
 
 export function CustomColorsSection(props: CustomColorsTypes) {
```

Comment:
'selectedPresets' has been removed from the props but it is still present in the types.

Reply:
I used this prop